package.json

```diff
@@ -50,10 +51,9 @@
     "lint:fix": "eslint -f unix 'src/**/*.ts' --fix",
     "prettier": "prettier --list-different 'src/**/*.ts'",
     "prettier:fix": "prettier --write 'src/**/*.ts'",
-    "test": "jest && npm run lint && npm run prettier"
+    "test": "tsc && jest && npm run lint && npm run prettier"
```

Comment:
--noEmit

Reply:
Since this is a library, it's not needed. The wallaby issue is limited to our services since serve-runtime is prioritizing build (if it exists) over src. I will add it anyway for consistency's sake.

Reply:
I just think avoid creating the build directory unnecessarily is good, it always seems to create random issues now and then